Formula/shapelib.rb

```diff
@@ -20,8 +21,11 @@ class Shapelib < Formula
   depends_on "cmake" => :build
 
   def install
-    system "cmake", ".", *std_cmake_args
-    system "make", "install"
+    args = std_cmake_args.reject { |s| s["CMAKE_INSTALL_LIBDIR"] }
```

Comment:
Could you add a comment explaining why we need to do this?  We have normally been using the `CMAKE_INSTALL_LIBDIR` flag to make sure things only get installed in `lib` and not `lib64`, but presumably that breaks something in this formula.

Reply:
Haven't taken exact look into CMake build setup but default `-DCMAKE_INSTALL_LIBDIR=lib` causes `*.so` files to be "installed" into `/tmp` build directory, so it might be interpreting it as local path.

Probably same reason as other formulae like:
- jpeg-turbo
- plplot
- mozjpeg
- lasi

Not sure why this only happens on Linux though.

Reply:
Added comment that the input is interpreted as relative to current running directory.

The original formula generates the following CMakeCache.txt
```
//install location for object code libraries
CMAKE_INSTALL_LIBDIR:PATH=/tmp/shapelib-20210807-36205-gc1wdl/shapelib-1.5.0/lib
```